test/mpi/util/mtest_thread.c

```diff
@@ -82,6 +82,12 @@ int MTest_thread_barrier_free(void)
     return MTest_thread_lock_free(&barrierLock);
 }
 
+#define CHECK_LOCK(err_)                                \
```

Comment:
`CHECK_LOCK` is a bit ambiguous. How about `LOCK_ERR_CHECK` or just `CHECK_ERR`?

Reply:
Sure, will update.